sites/hashdev/src/_pages/blog/7_block-paved-path.mdx

```diff
@@ -157,7 +157,7 @@ You are probably overwhelmed by that list, and so are all the developers who are
 
 <TalkSlide width="1362" height="766" src="/blog/7_block-paved-path/block-path-14.jpg">
 
-Third, there are incentive issues. It takes a lot of developer labour to add semantic markup to a website, and doesn't offer immediate benefits in return. Structured data can certainly improve SEO and content discovery, but beyond that we haven't found enough compelling use cases for it.
+Third, there are incentive issues. It takes a lot of developer labour to add semantic markup to a website, and doesn't offer immediate benefits in return. Structured data can certainly improve SEO and content discovery, but beyond that we haven't found enough compelling use cases for it [for creators and publishers].
```

Comment:
Is this supposed to be a link? I don't think the square brackets are needed otherwise

```suggestion
Third, there are incentive issues. It takes a lot of developer labour to add semantic markup to a website, and doesn't offer immediate benefits in return. Structured data can certainly improve SEO and content discovery, but beyond that we haven't found enough compelling use cases for it for creators and publishers.
```

Reply:
Not supposed to be a link, but an extraneous addition/clarification atop Maggie's actual talk.

Reply:
This doesn't seem different to the other additions/clarifications in the diff which aren't bracketed, so I'd lose it.